orangecontrib/timeseries/timeseries.py

```diff
@@ -106,6 +106,10 @@ def from_data_table(cls, table, time_attr=None):
                 and table.time_delta is not None:
             return table
 
+        if isinstance(table, Timeseries) and table.time_variable not in \
+                table.domain:
```

Comment:
It's nicer to line break on conjunctions.

Reply:
Does `x in table.domain` now cover metas? :sweat_smile:

Reply:
> Does x in table.domain now cover metas? 😅

Yes.